src/holofuel/components/layout/PrimaryLayout/PrimaryLayout.js

```diff
@@ -18,9 +18,9 @@ export function PrimaryLayout ({
   children,
   headerProps = {}
 }) {
-  const { data: { holofuelActionableTransactions: actionableTransactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'network-only' })
+  const { data: { holofuelActionableTransactions: actionableTransactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'cache-and-network' })
```

Comment:
Not quite sure what changed, such that this fetchPolicy now works, but I'm *very happy it does*!